xUnique.py

```diff
@@ -217,21 +229,12 @@ def sort_pbxproj(self, sort_pbx_by_file_name=False):
         pbx_flag = False
         last_two = deque([])
 
-        def file_dir_cmp(x, y):
-            if '.' in x:
-                if '.' in y:
-                    return cmp(x, y)
-                else:
-                    return 1
-            else:
-                if '.' in y:
-                    return -1
-                else:
-                    return cmp(x, y)
+        def file_dir_order(x):
+            return ('.' in x, x)
```

Comment:
The fact is `sorted` function in python3 doesn't accept cmp attribute and the challenge is emulate old sorting behavior only with key extract function